{{ cookiecutter.safe_formal_name }}/gradle.properties

```diff
@@ -15,8 +15,7 @@ org.gradle.jvmargs=-Xmx1536m
 # Android operating system, and which are packaged with your app's APK
 # https://developer.android.com/topic/libraries/support-library/androidx-rn
 android.useAndroidX=true
-# Do not automatically convert third-party libraries to use AndroidX, since
-# rubicon-java fails this conversion.
-android.enableJetifier=false
-# Ensure pythonlib is extracted from AAB bundles at runtime
-android.bundle.enableUncompressedNativeLibs=false
+# Automatically convert third-party libraries to use AndroidX
+android.enableJetifier=true
+# Kotlin code style for this project: "official" or "obsolete":
+kotlin.code.style=official
```

Comment:
Chaquopy doesn't require any of the previous changes to this file, so I've reverted it to the default version generated by Android Studio 4.2, which corresponds to the Android Gradle plugin version in build.gradle.